app/errors/error_service.ts

```diff
@@ -21,6 +21,10 @@ export class ErrorService {
       return;
     }
     alertService.error(String(error));
+    if (window.opener) {
+      console.log("Posting error message to " + window.location.origin);
+      window.opener?.postMessage(String(error), window.location.origin);
```

Comment:
```suggestion
      window.opener.postMessage(String(error), window.location.origin);
```

Reply:
I removed the log statements per Jim's suggestions, so the if became useless - so I kept the question mark